packages/vue/src/components/sign-up.vue

```diff
@@ -12,9 +11,8 @@ import AuthenticatorSignUpFormFields from './authenticator-sign-up-form-fields.v
 
 import { useAuthenticator } from '../composables/useAuth';
 
-const useAuthShared = createSharedComposable(useAuthenticator);
-const facadeValues = useAuthShared();
-const props = useAuthShared();
+const facadeValues = useAuthenticator();
+const props = useAuthenticator();
```

Comment:
Didn't want to go too far into refactoring (this needs changing the templates too), so didn't change code outside using the `sharedAuthenticator`.

Reply:
Makes sense, although doing a quick (and uninformed) glance through the file it does read as though we would be able to use either just `facade` or `props`